Makefile

```diff
@@ -102,8 +102,17 @@ eunit: couch
 
 .PHONY: javascript
 # target: javascript - Run JavaScript test suites or specific ones defined by suites option
-javascript: all share/www/script/test
-	@dev/run -q --with-admin-party-please test/javascript/run $(suites)
+javascript: all
+	@mkdir -p share/www/script/test
+ifeq ($(IN_RELEASE), true)
+	@cp test/javascript/tests/lorem*.txt share/www/test/
+else
+	@mkdir -p src/fauxton/dist/release/test
+	@cp test/javascript/tests/lorem*.txt src/fauxton/dist/release/test/
+endif
+	@ulimit -n 10240
```

Comment:
the ulimit? why doesn’t that work on FreeBSD? Different modifier?

The reason is that we are now using random db names and during the course of the test runs we open many hundreds of fds and things fail with emfile when this isn’t in here.